packages/tabs/tab.md

```diff
@@ -41,7 +41,10 @@ For tabs that have an icon and no visible label, the icon should have an `aria-l
 
 ```html
 <sp-tab value="complete" aria-label="Tab with checkmark">
-    <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
+    <sp-icon-checkmark
+        slot="icon"
+        label="Checking your work"
+    ></sp-icon-checkmark>
```

Comment:
@nikkimk would love some feedback on this one after Marissa's [feedback](https://github.com/adobe/spectrum-web-components/pull/5586#discussion_r2208270711)! Do we want it like this, or how it was previously, or something else?